app/editor/src/components/formik/checkbox/FormikCheckbox.tsx

```diff
@@ -22,7 +22,8 @@ export const FormikCheckbox = <T,>({
   const { values, errors, touched, handleBlur, handleChange, isSubmitting } = useFormikContext<T>();
 
   field = field ?? name;
-  const error = (errors as any)[field] && (touched as any)[field] && (errors as any)[field];
+  const errorMessage = getIn(errors, field);
```

Comment:
Fixed all of our formik components so they can handle fields that are part of an array.